doc/en/user/source/services/wfs/axis_order.rst

```diff
@@ -5,38 +5,56 @@ Axis ordering
 
 WFS 1.0.0 servers return geographic coordinates in longitude/latitude (x/y) order, the most common way to distribute data. For example, many shapefiles adopt this order by default. 
 
-However, the traditional axis order for geographic and cartographic systems is the opposite—latitude/longitude (y/x)—and the later WFS specifications respect this. The default axis ordering support is:
+However, the traditional axis order for geographic and cartographic systems is the opposite—latitude/longitude (y/x) — and the later WFS specifications respect this. The default axis ordering support is:
```

Comment:
I think something like:

However, the traditional axis order for geographic and cartographic systems is the opposite: latitude/longitude (y/x),  and the later WFS specifications respect this

Reply:
Thanks I will resolve it to have a similar structure as the proceeding sentence:

> However, the traditional axis order for geographic and cartographic systems in latitude/longitude (y/x) order, and the later WFS specifications respect this.